cisco_aci/datadog_checks/cisco_aci/cisco.py

```diff
@@ -24,10 +24,10 @@
 
 class CiscoACICheck(AgentCheck):
 
-    HTTP_CONFIG_REMAPPER = {'ssl_verify': {'name': 'tls_verify'}}
+    HTTP_CONFIG_REMAPPER = {'ssl_verify': {'name': 'tls_verify'}, 'pwd': {'name': 'password'}}
```

Comment:
Could you cover this in tests ?

Example: https://github.com/DataDog/integrations-core/blob/1bf1bcf5f4af4a54f16bf45ad76d55c4f364ce05/etcd/tests/test_integration.py#L162-L192

And maybe add a test for existing `ssl_verify` remapping :)